Chapters/4-4-Chapter26.md

```diff
@@ -0,0 +1,384 @@
+
+"The mornings," I said. "My mother used to sing in the morning too when we lived in the forest."
+
+"Thing's been singing and growing for at least three generations. Looks to be even older than that. Sort of a legend in town – Glad it survived."
+
+With the same gentleness she had shown him before, she took Jathi upon her back while I stared at the tree.
+
+"Oi. Come on," she beckoned.
+
+---
+
+The observatory was not as I expected it. There were no windows to be broken or walls to be split. It had been built into the ground, or perhaps the ground had been built around it. A set of three large metallic hatches protruded from the ground on the side facing Ara, closed to the elements. On the Ulu-side was a simple-looking wooden entryway with a wooden bell hung from a thick rope. Whether out of habit or the desire for some normalcy, Pomri jostled the wooden bell three times. Eager to rest, I lifted the latch and opened the door before she could finish her announcement of arrival.
+
+I found a warm area in middle of the room, where a carpet had been laid between two chairs, and sprawled out on the floor, removing my shawl and bag after.
+
+"I'm sooo tiiiired," I said, stretching my limbs. My heavy breathing was more noticeable in the quiet of the place.
+
+Pomri knealt down beside me and gently rested Jathi on the ground. She ventured to some dark corner of the room, as if distinctly familiar with this place, and returned with a pillow in her hands. Slowly, her hands raised Jathi's cold glowing body and gently rested it upon the pillow.
+
+"You can sleep later," she glanced at me with her voice. "Come, we need to look for the message."
+
+Her dark lantern rocked about with a subtle metallic creak as she rummaged through the place, moving quickly in the dark but never missing a step. A modest hearth with an empty pot took up the back side of the one room building. To the left and right was the rest of it, mostly instruments and tools and shelves full of records.
+
+"Ah!" she exclaimed, moving to a workbench. From a cabinet underneath, she pulled a large, thick, glass canister and popped a cork with her fingers. As she poured its contents into the lantern's upper tank, a distinctively floral aroma filled the room.
+
+"What _is_ that?" I asked.
+
+"Oil of Anomin. It's a thick oil from the bran of anomin grains. We use it to lubricate machines, and to do this..."
+
+With a firestarter just above the oil pool of her lantern, she lit the flame again and the area around her was instantly illuminated. Shadows of kanosin blue and a firey red were cast against the wood of the table and the floor, reflected on the water of her skin, swallowed by the deep black of the painted wall.
+
+"Right then. Let's find this message."
+
+I groaned in agony. She took a moment to glance at me, as if to hurry me without words.
+
+"Sorry, o supreme leader, but your nap will have to wait."
```

Comment:
lols kylo ren. this chapter keeps on reminding me of a bunch of unrelated things.